crates/sui/tests/cli_tests.rs

```diff
@@ -679,6 +683,32 @@ async fn test_move_call_args_linter_command() -> Result<(), anyhow::Error> {
 
     assert!(resp.is_err());
 
+    // Try a transfer with explicitly set gas price.
```

Comment:
Can we also have a different test where this has a large gas price?

Reply:
Done